redi-connect-front/src/components/organisms/RediHeroLanding.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import classnames from 'classnames'
+import { Section, Container, Element, Heading, Content, Columns } from 'react-bulma-components'
+import { useTranslation } from 'react-i18next'
+import Icon from '../atoms/Icon'
+import Image from '../atoms/Image'
+import './RediHeroLanding.scss'
+
+interface Props {
+  type: 'mentor' | 'mentee'
+}
+
+const RediHeroLanding = ({ type }: Props) => {
+  const { t } = useTranslation()
+  const programSteps: Array<{ name: string, image: any }> =
+    t(`loggedOutArea.homePage.hero.${type}.steps`, { returnObjects: true })
+  const baseClass = `${type}-border-color`
+
+  return (
+    <>
+      <Section className={classnames({ [`hero-landing--${type}`]: type })}>
```

Comment:
Main class is still a modifier.

Reply:
`section` element added to be consistent with the rest of the css